app/main/lib/text_similarity.py

```diff
@@ -109,7 +109,7 @@ def get_model_and_threshold(search_params):
   if 'per_model_threshold' in search_params and isinstance(search_params['per_model_threshold'], list) and [e for e in search_params['per_model_threshold'] if e["model"] == model_key]:
       threshold = [e for e in search_params['per_model_threshold'] if e["model"] == model_key][0]["value"]
   if threshold is None:
-      app.logger.error(
+      app.logger.info(
```

Comment:
This might be best as `warn` . It's a situation we don't expect to see from Check and would be good to track and eliminate if it does happen.

Reply:
I agree with @computermacgyver , this should be a `warn` since it may be actionable.